src/index.js

```diff
@@ -1,61 +1,65 @@
-let AWS = require('aws-sdk');
-let Entity = require('./modules/entity');
-let fs = require('fs');
-let log4js = require('log4js');
-let MicroCache = require('micro-cache');
-let Person = require('./modules/person');
-let util = require('util');
-
-let FileCertificate = {
+const AWS = require('aws-sdk');
+const fs = require('fs');
+const log4js = require('@log4js-node/log4js-api');
+const MicroCache = require('micro-cache');
+const _ = require('lodash');
+const util = require('util');
+const Person = require('./modules/person');
+const Entity = require('./modules/entity');
+
+const FileCertificate = {
   readCertificate: async (opts) => {
-    if (opts.cert === '' || opts.key === '' ||
-      !fs.existsSync(opts.cert) || !fs.existsSync(opts.key)) {
+    if (opts.cert === '' || opts.key === ''
+      // eslint-disable-next-line security/detect-non-literal-fs-filename
+      || !fs.existsSync(opts.cert) || !fs.existsSync(opts.key)) {
       throw new Error(`Client cert '${opts.cert}' or key '${opts.key}' can not be found`);
     }
 
     return {
-      cert:               fs.readFileSync(opts.cert),
-      key:                fs.readFileSync(opts.key),
-      rejectUnauthorized: false
+      // eslint-disable-next-line security/detect-non-literal-fs-filename
+      cert: fs.readFileSync(opts.cert),
+      // eslint-disable-next-line security/detect-non-literal-fs-filename
+      key: fs.readFileSync(opts.key),
+      rejectUnauthorized: false,
     };
-  }
+  },
 };
 
-let S3Certificate = {
+const S3Certificate = {
   readCertificate: async (opts) => {
-    let s3 = new AWS.S3();
-    let cert = await s3.getObject({
+    const s3 = new AWS.S3();
+    const cert = await s3.getObject({
       Bucket: opts.certBucket,
-      Key:    opts.certKey
+      Key: opts.certKey,
     }).promise().catch((err) => {
       throw Error('S3 get cert error', err);
     });
-    let key = await s3.getObject({
+    const key = await s3.getObject({
       Bucket: opts.keyBucket,
-      Key:    opts.keyKey
+      Key: opts.keyKey,
```

Comment:
keyKey!? lol just noticed it. I'm fine with it but it's a weird name 🔑 🔑

Reply:
Yes. It's the bucket key for the cert key.